spec/system/publishers_can_edit_a_draft_vacancy_as_a_school_spec.rb

```diff
@@ -3,26 +3,53 @@
 RSpec.describe "Publishers can edit a draft vacancy" do
   let(:publisher) { create(:publisher) }
   let(:school) { create(:school) }
-  let!(:vacancy) do
-    VacancyPresenter.new(build(:vacancy,
-                               job_title: "Draft vacancy",
-                               working_patterns: %w[full_time part_time],
-                               job_roles: %w[teacher]))
-  end
-  let(:draft_vacancy) { Vacancy.find_by(job_title: vacancy.job_title) }
 
   before { login_publisher(publisher: publisher, organisation: school) }
 
   context "editing an incomplete draft vacancy" do
-    let(:incomplete_vacancy) { create(:vacancy, :draft, job_roles: %w[teacher], salary: nil) }
+    context "when there are steps with errors" do
+      let(:vacancy_with_errors) do
+        create(:vacancy, :draft, job_roles: %w[teacher], salary: nil)
+      end
 
-    before { incomplete_vacancy.organisation_vacancies.create(organisation: school) }
+      before do
+        vacancy_with_errors.organisation_vacancies.create(organisation: school)
+        visit organisation_job_path(vacancy_with_errors.id)
+      end
 
-    scenario "cannot submit a incomplete draft from the manage job listing page" do
-      visit organisation_job_path(incomplete_vacancy.id)
+      scenario "cannot submit a incomplete draft from the manage job listing page" do
+        expect(page).to have_content(I18n.t("pay_package_errors.salary.blank"))
+        expect(page).to_not have_content(I18n.t("buttons.submit_job_listing"))
+      end
 
-      expect(page).to have_content(I18n.t("pay_package_errors.salary.blank"))
-      expect(page).to_not have_content(I18n.t("buttons.submit_job_listing"))
+      scenario "correct status tag is displayed" do
+        within(".review-component#pay_package") do
+          expect(page).to have_content(I18n.t("shared.status_tags.action_required"))
+        end
+      end
+    end
+
+    context "when a step has not been completed" do
```

Comment:
Can you make this description more precise? The ticket says the status tag is 'not started' *when no question has been answered* in that section.